packages/slate-dev-benchmark/src/index.js

```diff
@@ -0,0 +1,5 @@
+const { Repository, repo } = require('./components/Repository')
```

Comment:
We cannot unfortunately.  Because if we use babel, we must run benchmark with babel-node.  Then we our estimated time will include babel parsing time.